sale_automatic_workflow/data/automatic_workflow_data.xml

```diff
@@ -34,8 +33,8 @@
         >[('state', '=', 'draft'), ('posted_before', '=', False)]</field>
         <field name="user_id" ref="base.user_root" />
     </record>
-    <record id="automatic_workflow_sale_done_filter" model="ir.filters">
-        <field name="name">Automatic Workflow Sale Done Filter</field>
+    <record id="automatic_workflow_sale_lock_filter" model="ir.filters">
```

Comment:
Please don't rename XML-ID records if there's no need. It's OK to rename the text for better clarity, but the technical XML-ID is OK (the value is still `done`), and you avoid potential compatibility/migration problems.